conda-store-server/conda_store_server/server/templates/create.html

```diff
@@ -26,6 +26,8 @@
         <label for="specification">Specification:</label>
         <div id="editor" style="height: 400px;">{{ specification or "" }}</div>
         <textarea name="specification" id="specification" style="display: none;"></textarea>
+        <input type="checkbox" name="is-lockfile" id="is-lockfile" />
+        <label for="is-lockfile">This specification contains a <a href="https://github.com/conda-incubator/conda-lock">Conda-Lock</a> lockfile</label>
```

Comment:
I don't like this checkbox. This should be a dropdown with all possible inputs (more are possibly coming, like `explicit.txt` lockfiles). For now: `environment.yml`, `conda-lock.yml` are the options.